### DISCUSSION TITLE
When will the official 2.0 release be launched?

### DISCUSSION THREAD

**Discussion by FrancisKaneY** [tauri-apps/tauri#10419] [General]
I am very interested in using Taurus 2.0 in my project, but I don't know when the stable version of the project will be released. Is there a plan to release a stable version by the end of this year?

**Comment by FabianLars** [comment]
Yes it will be released this year.

**Comment by charlesschaefer** [comment]
Hi @FrancisKaneY, accordingly to the recent blog post https://v2.tauri.app/blog/tauri-2-0-0-release-candidate/, it will be release in the end of August, this year.

**Comment by kira-coding** [comment]
It is exciting to wait and so. But is it really near.  
estimations

**Comment by yanbowe** [comment]
It's September, when will the official version be released
